src/partner-access/partner-access.service.ts

```diff
@@ -67,33 +73,21 @@ export class PartnerAccessService {
   async validatePartnerAccessCode(
     partnerAccessCode: string,
   ): Promise<{ status: PartnerAccessCodeStatusEnum }> {
-    const PartnerAccessCodeStatusEnum = await this.checkCodeStatus(partnerAccessCode);
+    await this.getValidPartnerAccessCode(partnerAccessCode);
     return {
-      status: PartnerAccessCodeStatusEnum,
+      status: PartnerAccessCodeStatusEnum.VALID,
     };
   }
 
   async updatePartnerAccessCodeUser(
```

Comment:
i don't think this is being called anywhere now right? does it make sense to remove?